apps/frontend/src/lib/oauth/github.ts

```diff
@@ -1,31 +1,42 @@
 import {
   ChipIssuer,
   LeaderboardEntryType,
-  UpdateLeaderboardEntryRequest
+  UpdateLeaderboardEntryRequest,
 } from "@types";
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function MapGithubCommitContributionsToLeaderboardEntry(authToken: string, chipIssuer: ChipIssuer, resp: any): UpdateLeaderboardEntryRequest {
-
+export function MapGithubCommitContributionsToLeaderboardEntry(
+  authToken: string,
+  chipIssuer: ChipIssuer,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  resp: any
+): UpdateLeaderboardEntryRequest {
   let commits: number = 0;
 
-  if (resp && resp.data && resp.data.user && resp.data.user.contributionsCollection && resp.data.user.contributionsCollection.totalCommitContributions) {
+  if (resp?.data?.user?.contributionsCollection?.totalCommitContributions) {
     commits += resp.data.user.contributionsCollection.totalCommitContributions;
   }
-  if (resp && resp.data && resp.data.user && resp.data.user.contributionsCollection && resp.data.user.contributionsCollection.totalIssueContributions) {
+  if (resp?.data?.user?.contributionsCollection?.totalIssueContributions) {
     commits += resp.data.user.contributionsCollection.totalIssueContributions;
   }
-  if (resp && resp.data && resp.data.user && resp.data.user.contributionsCollection && resp.data.user.contributionsCollection.totalPullRequestContributions) {
-    commits += resp.data.user.contributionsCollection.totalPullRequestContributions;
+  if (
+    resp?.data?.user?.contributionsCollection?.totalPullRequestContributions
+  ) {
+    commits +=
+      resp.data.user.contributionsCollection.totalPullRequestContributions;
   }
-  if (resp && resp.data && resp.data.user && resp.data.user.contributionsCollection && resp.data.user.contributionsCollection.totalPullRequestReviewContributions) {
-    commits += resp.data.user.contributionsCollection.totalPullRequestReviewContributions;
+  if (
+    resp?.data?.user?.contributionsCollection
+      ?.totalPullRequestReviewContributions
+  ) {
+    commits +=
+      resp.data.user.contributionsCollection
+        .totalPullRequestReviewContributions;
```

Comment:
I'm so sorry you had to clean up this crap

Reply:
haha no worries